test/extended/util/annotate/rules.go

```diff
@@ -304,6 +304,11 @@ var (
 			`Unidling should work with TCP`,
 			`Unidling should handle many TCP connections`,
 		},
+		// tests that don't pass under Cilium
+		"[Skipped:Network/Cilium]": {
```

Comment:
This needs to be clearer about why this section is here:
- We do not support Cilium, but we run e2e tests against it because ... why?
  - (Including "why don't we run e2e tests against other third-party plugins?")
- More info about the e2e Cilium test runs can be found at ... ?
- There are no links to BZ/Jira for the test skips here because we do not intend to fix these bugs ourselves because we don't support this configuration. (But, for some reason, we still need to run e2e tests against it?)

Reply:
as per this documentation https://docs.cilium.io/en/v1.12/gettingstarted/k8s-install-openshift-okd/ and https://isovalent.com/blog/post/2022-03-openshift and OCP still supports cilium as third party cni solution and this is evident from [4.11 CI](https://prow.ci.openshift.org/job-history/gs/origin-ci-test/logs/periodic-ci-openshift-release-master-ci-4.11-e2e-aws-cilium) runs as well.
We don't have BZ/Jira issue for this as it comes under CI stabilization work which is found while investigating [OCPVE-133](https://issues.redhat.com/browse/OCPVE-133).

Reply:
Being certified to run with OpenShift does not mean "_Red Hat_ tests that this plugin works with OpenShift", it means "_the vendor of this plugin_ tests that (specific versions of) this plugin work with (specific versions of) OpenShift" (plus stuff with support contracts and stuff).

> OCP still supports cilium as third party cni solution and this is evident from 4.11 CI runs as well.

In fact, no version of Cilium is certified against OCP 4.11. (https://access.redhat.com/articles/5436171). At any rate, if it was, that would be entirely independent of this CI job. Third party CNI plugin certification is done by having the third-party CNI plugin vendor run the e2e tests themselves and sending us the results.

> We don't have BZ/Jira issue for this as it comes under CI stabilization work which is found while investigating [OCPVE-133](https://issues.redhat.com/browse/OCPVE-133).

If there is no good reason for this job to exist, then the way to improve overall CI pass rate would be to just remove that job (or at least, remove it from the set of jobs that are counted in the overall pass rate.)

Reply:
@danwinship Thanks for the info, If OpenShift doesn't intend to run its e2e tests anymore with Cilium, then we can just remove that job from running. Let me work on a separate PR for this.